src/modules/tools/zprobe/ZProbe.cpp

```diff
@@ -240,16 +247,56 @@ bool ZProbe::return_probe(int steps)
     return true;
 }
 
-bool ZProbe::doProbeAt(int &steps, float x, float y)
+bool ZProbe::doProbeAt(int &steps, float x, float y, float actuator_positions[], int repeats)
```

Comment:
I'm not so sure that it is that easy to recover raw actuator positions for the trigger point with the current probe code. Could one guarantee that ```cartesian_to_actuator({x, y, z - steps_to_mm(steps)})``` always returns exact the same result as the above for all possible compensationTransforms, all different and future arm-solutions etc? In my opinion, actuator_position at probe trigger is the most fundamental datapoint and from that it is easy to recover the others.